src/leo_redundant_manager_api.erl

```diff
@@ -581,6 +581,19 @@ get_redundancies_by_key(Method, Key) ->
     AddrId = leo_redundant_manager_chash:vnode_id(BitOfRing, Key),
     get_redundancies_by_addr_id_1(ring_table(Method), AddrId, Options).
 
+-spec(get_redundancies_by_key(Method, Key, ConsistencyLevel) ->
+             {ok, #redundancies{}} |
+             {error, any()} when Method::method(),
+                                 Key::binary(),
+                                 ConsistencyLevel::[{ConsistencyItem, ConsistencyValue}],
+                                 ConsistencyItem::consistency_item(),
+                                 ConsistencyValue::non_neg_integer()).
```

Comment:
From what I understand, s/non_neg_integer/pos_integer/ would be more precise as each value can not be 0.